app/jinja_filters.py

```diff
@@ -134,7 +155,7 @@ def get_format_date(value):
 
 @pass_eval_context  # type: ignore
 @blueprint.app_template_filter()
-def format_datetime(context, date_time):
+def format_datetime(context: nodes.EvalContext, date_time: datetime) -> str:
```

Comment:
This should have the same return value as the `markup_safe` method because that is what it returns.